src/daria/image/image.py

```diff
@@ -573,3 +615,72 @@ def toGreen(self, return_image: bool = False) -> Optional[da.Image]:
                     "Only RGB or BGR images can be converted to green at the moment"
                 )
             self.metadata["color_space"] = "GREEN"
+
+    def toHue(self, return_image: bool = False) -> Optional[da.Image]:
+        """
+        Returns a hue channel version of the daria image.
+
+        Args:
+            return_image (bool): flag controlling whether the converted image
+                is returned, or converted internally.
+
+        Returns:
+            daria.Image (optional): converted image, if requested
+        """
+        # Only for RGB images for now.
+        assert self.metadata["color_space"] == "RGB"
```

Comment:
I guess one could add a toRGB(), or similar, to allow for at least BGR images also. But one could of course do this outside as well.

Reply:
I did not want to make it too complicated at this stage, but I thought something similar.